src/app/shared/store/navigation.state.ts

```diff
@@ -1,29 +1,34 @@
 import { Navigation } from './../models/navigation.model';
 import { State,Action,StateContext,Selector } from "@ngxs/store";
 import { Injectable } from '@angular/core';
-import { AddNavPath, DeleteNavPath, RemoveLastNavPath } from './navigation.actions';
+import { AddNavPath, SidenavToggle, DeleteNavPath, RemoveLastNavPath } from './navigation.actions';
 
 export interface NavStateModel {
   navigation: Navigation[];
+  sidenavOpen: boolean;
 }
 
 @State<NavStateModel>({
   name:'navigation',
   defaults: {
-    navigation:[]
+    navigation: [],
+    sidenavOpen: false
   }
 })
 
 @Injectable()
 export class NavigationState {
 
   @Selector()
-  static navigationPaths(state: NavStateModel): Navigation[] {return state.navigation}
+  static navigationPaths(state: NavStateModel): Navigation[] { return state.navigation }
+
+  @Selector()
+  static sidenavOpenTrue(state: NavStateModel): boolean {return state.sidenavOpen}
 
   @Selector()
   static navigationPathsMobile(state: NavStateModel): Navigation[] {
      const navigation = [...state.navigation]
-     return [navigation.pop()] 
+     return [navigation.pop()]
```

Comment:
please put semicolon where they are missed

Reply:
don't know why it showed up here, but it's not my changes
anyway, semicolon added